### PR TITLE
README Revamp: Addressing Issues #3 and #16

### PR DESCRIPTION
**Pull request by matthiasjrichter**
# Description
- Complete rewrite of README.md in preparation of introducing the project to a broader audience at the Social Anxiety Meetup.
- Enables easier addition of new *good first issues*.
- Addresses Issues  #3 and #16 but does not close them because further updates will be necessary.
- Depends on `docs/CONTRIBUTING.md`  for contribution guidelines.

## Type of change

This change is a documentation update.

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] *Not applicable*: ~~I have added tests that prove my fix is effective or that my feature works~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,41 @@
...
 - Git
+- HTML
+- CSS
+
+## Contributors
+<!-- List in order of appearance. Up for discussion. -->
+We appreciate the contributions from the following developers:
+- https://github.com/matthiasjrichter
+- https://github.com/patdel0
+- https://github.com/anniecodes404
```
Comment (patdel0) [inline README.md:38]: Whenever we get hundreds of contributors [extremely unlikely] and the list gets too long, we could do something like this: https://github.com/Significant-Gravitas/AutoGPT/blob/master/README.md#-contributors

File: README.md [hunk 2]
```diff
@@ -1,17 +1,41 @@
...
+## Contributing
+Please refer to our [contribution guide](docs/CONTRIBUTING.md) for further information.
+
+## Tech Used
 - Git
+- HTML
+- CSS
+
+## Contributors
+<!-- List in order of appearance. Up for discussion. -->
```
Comment (patdel0) [inline README.md:34]: Today I learned: we can add comments to markdown

**Review by patdel0 — APPROVED** [review]
This is fire 🔥

Excellent work putting this together!

File: README.md (hunk 1 above)
Comment (matthiasjrichter) [reply README.md:38]: Good suggestion! I've been looking into similar options.
I am deliberately keeping some things open to

1. Facilitate _good first issues_
2. Get things moving

**Pull request merged**